Snakefile

```diff
@@ -372,15 +375,40 @@ def get_dataset_label(wildcards):
     dataset = parts[0]
     return dataset
 
-# Run evaluation code for a specific dataset's pathway outputs against its paired gold standard
-rule evaluation:
+# Returns all pathways for a specific dataset
+def collect_pathways_per_dataset(wildcards):
+    dataset_label = get_dataset_label(wildcards)
+    return expand('{out_dir}{sep}{dataset_label}-{algorithm_params}{sep}pathway.txt', out_dir=out_dir, sep=SEP, algorithm_params=algorithms_with_params, dataset_label=dataset_label)
+
+# Run precision and recall for all pathway outputs for a dataset against its paired gold standard
+rule evaluation_pr_per_pathways:
```

Comment:
Is this now the behavior you switched to in #212

Reply:
No this one is slightly different. This will give each precision and recall for each output subnetwork/pathway. The one in 212 will use the frequencies of the nodes from all of the outputs per algorithm and create PR curves using that.